package.json

```diff
@@ -22,6 +22,7 @@
         "test:ci": "yarn test --no-cache --coverage --runInBand",
         "test:integration": "nx sauce @lwc/integration-tests",
         "test:performance": "nx test @lwc/perf-benchmarks",
+        "test:performance:best": "cd packages/@lwc/perf-benchmarks && yarn test:best",
```

Comment:
You might wonder "why not `nx test`"? And the answer is: it doesn't seem to set the `cwd` property, so Best can't find its `best.config.js` file, so it fails. 🤷

Reply:
We could move the `best.config.js` to the root of the project, but I like keeping Benchmark code isolated to the Benchmark packages.

Reply:
Hm well actually it turns out Best has a `--config` option we could use, but the problem then is that using `nx test @lwc/perf-benchmarks test:best` then causes NX to think that it needs to run the Tachometer tests in order to run the Best tests... maybe we should just keep this `cd` for now. 😛

Reply:
> it doesn't seem to set the `cwd` property, so Best can't find its `best.config.js` file, so it fails. 🤷

It's not the experience I have. I was surprised by this so I ran a couple of tests locally:
- Adding a `script` entry in one of the packages with `echo $PWD`: prints the correct package directory (not the root directory).
- Adding a `script` entry in one of the packages with `node src/index.js` and create an `index.js` with `console.log(process.cwd())`: same observation.

Reply:
I figured it out; see my other comment.